_inc/client/setup-wizard/intro-page/index.jsx

```diff
@@ -38,10 +38,10 @@ const IntroPage = props => {
 					{ __( 'What will %(siteTitle)s be used for?', { args: { siteTitle: props.siteTitle } } ) }
 				</h2>
 				<div className="jp-setup-wizard-answer-buttons">
-					<Button primary className="jp-setup-wizard-button">
+					<Button href="#/setup/income" primary className="jp-setup-wizard-button">
```

Comment:
We'll most likely want to add tracking to those links so we can see how people navigate through the wizard.

Reply:
I plan to add Tracks events across the entire setup wizard in one PR, after all the pages are merged. That way all the new events will be in one place and I can make a P2 to get design and marketing review on all the event names and props.